tests/test_normalize_to_span.py

```diff
@@ -0,0 +1,54 @@
+import spacy
+from spacy.tokens import Span
+from relationextraction import SpacyRelationExtractor
+from collections import Counter
+from heads_extract_component import HeadwordsExtraction
+
+
+def test_normalize_to_span():
+
+    nlp = spacy.load("en_core_web_lg")
+    nlp.add_pipe("heads_extraction", config={"force" = True})
+
+    doc = nlp("Mette Frederiksen is the Danish politician.")
+
+    normalized_token = doc[0]._.to_span  # Token
+
+    assert isinstance(normalized_token, Span)
+    assert normalized_token.text == "Mette"
+
+    normalized_token = doc[0:2]._.to_span  # Span
+
+    assert isinstance(normalized_token, Span)
+    assert normalized_token.text == "Mette Frederiksen"
+
+    normalized_token = doc[:]._.to_span  # Doc
```

Comment:
```suggestion
    normalized_token = doc._.to_span  # Doc
```
Didn't we already change this?

Reply:
not sure, probably in another test.